core/whistle_number_manager-1.0.0/src/wnm_number_crawler.erl

```diff
@@ -36,6 +36,10 @@
         ,whapps_config:get_integer(?WNM_CONFIG_CAT, <<"deleted_expiry_d">>, 90)
        ).
 
+-define(NUMBERS_TO_CRAWL
+        ,whapps_config:get_integer(?WNM_CONFIG_CAT, <<"number_of_documents_to_crawl">>, 1000)
```

Comment:
@lazedo 
whistle_couch has a property default_chunk_size which I believe we have been using elsewhere.  I would prefer this as I don't see much value in changing the chunk size for individual operations.... The intent is to get chunks proportional to the size of the db cluster, so its very convenient to change it in one place...